db/migrate/20210225152418_remove_index_on_task_name.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class RemoveIndexOnTaskName < ActiveRecord::Migration[6.1]
+  def up
+    change_table(:maintenance_tasks_runs) do |t|
```

Comment:
Any reason not to use `remove_index` and `add_index` directly?

Reply:
The only reason I could see using this over `remove_index` and `add_index` directly is that it makes it easier to change it to an LHM:

```ruby
class RemoveIndexOnTaskName < ActiveRecord::Migration[6.1]
  def up
    change_table(:maintenance_tasks_runs) do |t|
      t.remove_index(:task_name)
    end
  end

  def down
    change_table(:maintenance_tasks_runs) do |t|
      t.index(:task_name)
    end
  end
end
```

```ruby
require 'lhm'

class RemoveIndexOnTaskName < ActiveRecord::Migration[6.1]
  def up
    Lhm.change_table :maintenance_tasks_runs do |t|
      t.remove_index(:task_name)
    end
  end

  def down
    Lhm.change_table :maintenance_tasks_runs do |t|
      t.index(:task_name)
    end
  end
end
```

Not sure if if this is something we care about that much though 😄

Reply:
Yup it's really nice if it lets people adapt it more easily to LHM.